### ISSUE TITLE
Label orientation/position around node

### ISSUE THREAD

**Issue by kmalyavina** [visjs/vis-network#365]
I would like to be able to put the label to the right of a node. My use case is adding a legend to the graph. Label-centered-at-the-bottom gets quickly takes up a lot of vertical space and gets hard to follow when there are a lot of groups to show. 

I've tried a number of ways with margin, font alignment, circle-physics-off-with-small-size but could not get the text to line up properly. 

Idealy, you would be able to choose top/bottom/left/right for the position of the node label (in an attribute like "labelPosition")

Current -> Desired (example from OrientDB Studio)
<img width="349" alt="Screen Shot 2020-01-20 at 8 25 19 AM" src="https://user-images.githubusercontent.com/22348821/72729987-6f63d680-3b5e-11ea-8b30-876c56143dbf.png">

**Comment by Thomaash** [comment]
Hi @kmalyavina,

would you have the time to work on this?

**Comment by kmalyavina** [comment]
> Hi @kmalyavina,
> 
> would you have the time to work on this?

I'm not very familiar with the codebase, but I might be able to try implementing it if given some guidance.

**Comment by Thomaash** [comment]
That would be great. Feel free to open a draft PR once you have anything even if it doesn't work (it's much easier to offer guidance while looking at the code).

The place to start are the files `lib/network/modules/components/Node.js` which invokes the label rendering and maintains info about it's shape and position and `lib/network/modules/components/shared/Label.js` which renders the label itself.
